core/src/main/java/software/amazon/awssdk/retry/SimpleRetryPolicy.java

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright 2010-2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package software.amazon.awssdk.retry;
+
+import java.time.Duration;
+import software.amazon.awssdk.annotation.SdkPublicApi;
+import software.amazon.awssdk.retry.backoff.BackoffStrategy;
+import software.amazon.awssdk.retry.conditions.AndRetryCondition;
+import software.amazon.awssdk.retry.conditions.MaxNumberOfRetriesCondition;
+import software.amazon.awssdk.retry.conditions.RetryCondition;
+
+@SdkPublicApi
+public final class SimpleRetryPolicy implements RetryPolicy {
```

Comment:
Could this be private and exposed via the RetryPolicy.builder() method, like many of our other objects?